extensions/msal-node-extensions/package.json

```diff
@@ -38,7 +38,7 @@
   "module": "dist/test.esm.js",
   "dependencies": {
     "@azure/msal-common": "^5.0.0",
-    "bindings": "^1.5.0",
+    "bindings": "git://github.com/samuelkubai/node-bindings.git#v1.6.0",
```

Comment:
Did we follow up on creating an azure repo for this?

Reply:
Yes we did, we realized it would take a lot longer to process and thus decided to go on with this setup for the time being as we research on how best to switch to a repo under AzureAD